build/Dockerfile.alpine

```diff
@@ -9,36 +9,29 @@ RUN set -eux \
 
 RUN wget "https://static.rust-lang.org/rustup/dist/x86_64-unknown-linux-musl/rustup-init"; \
     chmod +x rustup-init; \
-    ./rustup-init -y --no-modify-path --default-toolchain 1.60.0; \
+    ./rustup-init -y --no-modify-path --default-toolchain 1.73.0; \
     rm rustup-init; \
     chmod -R a+w $RUSTUP_HOME $CARGO_HOME;
 
 # prepare go cache dirs
 RUN mkdir -p /.cache/go-build
 RUN chmod -R 777 /.cache
 
-# pre-fetch many deps
-WORKDIR /scratch
-COPY go-owasm /scratch/go-owasm
-RUN cd go-owasm/libgo_owasm && cargo fetch
-
 # allow non-root user to download more deps later
 RUN chmod -R 777 /usr/local/cargo
 
 # cleanup
 WORKDIR /code
-RUN rm -rf /scratch
 
 # add musl Rust targets
 RUN rustup target add x86_64-unknown-linux-musl
 
 # copy build scripts
-COPY go-owasm/build/build_alpine.sh /opt
-RUN chmod +x /opt/build*
+COPY build/*.sh /usr/local/bin/
+RUN chmod +x /usr/local/bin/*.sh
 
-# add config cargo
 RUN mkdir /.cargo
 RUN chmod +rx /.cargo
-COPY go-owasm/build/cargo-config /.cargo/config
+COPY build/cargo-config /.cargo/config
 
-CMD ["/opt/build_alpine.sh"]
+CMD ["bash", "-c", "echo 'Argument missing. Pass one build script (e.g. build_alpine.sh) to docker run' && exit 1"]
```

Comment:
this cmd run the receive script and output error if there is no script, right? Am I understand it correctly

Reply:
yes, correct.